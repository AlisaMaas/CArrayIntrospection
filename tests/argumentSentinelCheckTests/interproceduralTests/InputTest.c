#include "headers.h"

int bar (char *f, char *i, char *s, int l) {
    return foo(f, i, s, l);
}

void baz(char *d, char *s, int l) {
    memcpy(d, s, l);
}