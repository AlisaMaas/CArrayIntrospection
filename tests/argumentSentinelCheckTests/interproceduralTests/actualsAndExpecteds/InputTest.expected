bar with argument 0 should be annotated Fixed length of 7.
bar with argument 2 should be annotated Parameter length of 3.
baz with argument 1 should be annotated Parameter length of 2.
